From af93db050e56dd6cc99674c58c48d45fd2eb92ba Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Thu, 29 May 2014 18:22:04 +0200
Subject: [PATCH] Fix desktop file template

warning: value "GTK;Application;Utility;" for key "Categories" in group "Desktop Entry" contains a deprecated value "Application"
---
 sm.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sm.desktop.in b/sm.desktop.in
index f37e974..4b6033d 100644
--- a/sm.desktop.in
+++ b/sm.desktop.in
@@ -5,4 +5,4 @@ Exec=@BINPATH@/sm
 Icon=sm
 Terminal=false
 Type=Application
-Categories=GTK;Application;Utility;
+Categories=GTK;Utility;
-- 
1.9.3

